packages/jest-config-terra/package.json

```diff
@@ -39,7 +39,10 @@
     "babel-jest": "^26.6.3",
     "identity-obj-proxy": "^3.0.0",
     "jest-environment-jsdom": "26.6.2",
-    "jest-mock": "^26.6.2"
+    "jest-mock": "^26.6.2",
+    "@cerner/terra-cli": "^1.7.0",
```

Comment:
What do we uses terra-cli for here? Can we not use it?

Reply:
using it for `Logger`

Reply:
Can you move @cerner/terra-cli and "@jest/reporters" to the top of the dependency list? Typically scoped dependencies with the @ are listed first alphabetically.